bin/update_aws_spec.py

```diff
@@ -0,0 +1,30 @@
+import os
+from pathlib import Path
+
+import yaml
+
+from localstack.utils.openapi import get_localstack_openapi_spec
+from localstack.version import version as ls_version
+
+import click
+
+@click.command()
+@click.option('--latest', is_flag=True, default=False, help='If enabled, sets the version of the spec to latest.')
+def update_aws_spec(latest: bool) -> None:
+    """
+    This function retrieves the OpenAPI specs declared via plugins, sets a given version and store the result
+    in a file at a given path.
+    """
+    version = "latest" if latest else ls_version
+    openapi_path = Path(os.path.dirname(__file__)) / ".." / "openapi" / "emulators"/ f"localstack-spec-{version}.yml"
```

Comment:
Q: What is the motivation behind adding the version to the filename? I feel like the version field in the spec together with tags and releases should be enough, and with different file names diffing just gets more complicated... I would love to be able to see the differences between two versions of the specs just by looking at the changes in the commits. :)

Reply:
My initial idea was to keep here all the tagged specs plus a `latest`. But since the specs are actually attached to the release, I do agree that this is redundant.